course-v2/layouts/_default/baseof.html

```diff
@@ -42,6 +42,16 @@
               <div class="d-flex">
                 <div class="col-12 p-0" id="main-course-section">
                   <div class="card-body">
+                    {{ if not $isCourseHomePage }}
+                      <button 
+                        type="button" 
+                        id="show-desktop-course-drawer" 
+                        class="show-course-info-left-arrow large-and-above-only"
+                        aria-label="Show Course Info"
+                        >
+                        <img class="left-arrow" src="/images/left_arrow.svg" alt=""/>
```

Comment:
I was able to drag over and highlight the x icon, which looks a little odd:
![image](https://user-images.githubusercontent.com/12089658/194164591-0de3ead0-98d1-4fae-8fd9-17b9bdc45369.png)

Can we prevent this from happening with the `user-select: none;` CSS rule?

Reply:
Since we are replacing the div wrappers with buttons so now this should be fine